.vale/styles/Microsoft/Accessibility.yml

```diff
@@ -0,0 +1,25 @@
+extends: existence
```

Comment:
Do we need to add all of these files to the repository? e.g. I don't see these files in the Kedro repo - does it work differently there? or are these files just not being added to the repository and ignored?

Reply:
They're in a different location in the Kedro repo because they've put them in the `.github` actions folder: https://github.com/kedro-org/kedro/tree/main/.github/styles/Kedro

I've also added more for Vizro because I worked bottom up for Kedro, and added in Styles that I thought would help.  There was so much existing documentation that I had to be quite light on the guidelines I enforced.

For Vizro I started with the full set of Microsoft styles (plus added some others for readability) and just trimmed or removed those I felt unnecessary. With less documentation in the project right now, it's easier to fix things to adhere to best practice, which was something impossible for Kedro.

Vizro Vale is going to be tougher to pass than Kedro once this is up and running, which is important since there are fewer of us working on docs. We need all the automation we can get!

Reply:
Got it! Thanks for clarifying! 👍